File: aat-android/src/main/java/ch/bailu/aat/util/ui/UiTheme.java
package ch.bailu.aat.util.ui;

import android.view.View;
import android.widget.ListView;
import android.widget.TextView;

public interface UiTheme {

    float HEADER_TEXT_SIZE = 18f;

    void background(View v);
    void backgroundAlt(View v);

    void button(View v);

    void topic(TextView v);
    void header(TextView v);

    void content(TextView v);
    void contentAlt(TextView v);

    void toolTip(TextView v);

    int getBackgroundColor();
    int getHighlightColor();

    int getGraphBackgroundColor();
    int getGraphLineColor();
    int getGraphTextColor();

    void list(ListView l);



}

File: aat-android/src/main/kotlin/ch/bailu/aat/util/ui/theme/UiTheme.kt
package ch.bailu.aat.util.ui.theme

import android.view.View
import android.widget.ListView
import android.widget.TextView

interface UiTheme {
    fun background(v: View)
    fun backgroundAlt(v: View)
    fun button(v: View)
    fun topic(v: TextView)
    fun header(v: TextView)
    fun content(v: TextView)
    fun contentAlt(v: TextView)
    fun toolTip(v: TextView)
    fun getBackgroundColor(): Int
    fun getHighlightColor(): Int
    fun getGraphBackgroundColor(): Int
    fun getGraphLineColor(): Int
    fun getGraphTextColor(): Int
    fun list(l: ListView)

    companion object {
        const val HEADER_TEXT_SIZE = 18f
    }
}
